### PR TITLE
OpenWebNet automatic scan of automation, lighting and temperature control devices.

### PR DESCRIPTION
**Pull request by Spugna85**
Implementation of automatic scan of automation, lighting and temperature
control.
This include automatic insert of any automation, lighting and temperature control devices.

Other bug fix....

### CODE REVIEW

File: main/RFXNames.cpp [hunk 1]
```diff
@@ -498,6 +498,7 @@ const char *RFX_Type_SubType_Desc(const unsigned char dType, const unsigned char
 		{ pTypeTEMP, sTypeTEMP9, "RUBiCSON" },
 		{ pTypeTEMP, sTypeTEMP10, "TFA 30.3133" },
 		{ pTypeTEMP, sTypeTEMP11, "WT0122 Pool sensor" },
+		{ pTypeTEMP, sTypeTEMP12, "Legrand MyHome Temp" },
```
Comment (gizmocuz) [inline main/RFXNames.cpp]: You can not do this, just use any of the existing temp sensors, no need for a 12th
this is only for rfxcom, please use
pTypeGeneral/sTypeTemperature

File: main/RFXtrx.h [hunk 2]
```diff
@@ -8,11 +8,11 @@
 //
 
 /*
-                                                                   
+
```
Comment (gizmocuz) [inline main/RFXtrx.h]: You can not touch this file, it is not ours

File: main/RFXtrx.h [hunk 3]
```diff
@@ -1673,7 +1674,7 @@ typedef union tRBUF {
 		BYTE	tempsign : 1;
 #endif
 		BYTE	temperaturel;
-		BYTE	humidity; 
+		BYTE	humidity;
```
Comment (gizmocuz) [inline main/RFXtrx.h]: Also turn of the formatting option in your editor

File: www/app/LightsController.js [hunk 4]
```diff
@@ -3482,9 +3482,17 @@ define(['app'], function (app) {
 					$("#dialog-addmanuallightdevice #fanparams #combocmd3 option:selected").text();
 				mParams+="&id="+ID;
 			}
-			else if ((lighttype==305) || (lighttype==306)) {
-				//OpenWebNet Blinds/light
-				var ID="OpenWebNet";
+			else if (lighttype==305) {
```
Comment (gizmocuz) [inline www/app/LightsController.js:3485]: you test for 305 now only, in lights.html you removed 305 and 306, is this going OK?

**Comment by Spugna85** [comment]
can you merge this? is there any problem in this request?

**Comment by gizmocuz** [comment]
Maybe I asked a question above?

**Comment by Spugna85** [comment]
I fixed all in the second commit.

**Comment by gizmocuz** [comment]
Thanks!

**Pull request merged**